int64[] x_pixel
int64[] y_pixel
---
float64[] x_world
float64[] y_world